frontend-cert/css-projects/css-colors/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Colors Lab</title>
```

Comment:
I would call this something colored boxes

```suggestion
    <title>Colored Boxes</title>
```